src/fidesops/schemas/connection_configuration/connection_secrets_saas.py

```diff
@@ -58,8 +58,11 @@ def get_saas_schema(self) -> Type[SaaSSchema]:
                 if connector_param.default_value
                 else (str, ...)
             )
-        return create_model(
+        SaaSSchema.__doc__ = f"{self.saas_config.type.capitalize()} secrets schema"  # Dynamically override the docstring
```

Comment:
Ooo this is so cool! How does this new docstring manifest? Is it more for logs?

Reply:
The docstring is automatically returned as the "description" in this new API endpoint:

```json
{
    "title": "sentry_connector_schema",
    "description": "Sentry secrets schema", <---
    "type": "object",
    "properties": {
        "access_token": {
            "title": "Access Token",
            "type": "string"
        },
        "domain": {
            "title": "Domain",
            "default": "sentry.io",
            "type": "string"
        }
    },
    "required": [
        "access_token"
    ],
    "additionalProperties": false
}
```

I don't think we'll use it in the front end, but I wanted to override the base class docstring which is: 
```
    Abstract base schema for updating SaaS connection configuration secrets.
    Fields are added during runtime based on the connector_params in the
    passed in saas_config"""
```

Reply:
This makes sense to me, thanks!

Reply:
great, thanks for checking!